src/compiler/diagnosticMessages.json

```diff
@@ -3310,6 +3310,10 @@
         "category": "Error",
         "code": 7034
     },
+    "Try `npm install @types/{0}` if it exists or adding `declare module '{0}'` in a separate file.": {

```

Comment:
Is "separate file" descriptive enough? Can we go with "a global declaration (d.ts) file" or something?

I might also want a pointer as to where to put the file but I can't think of a concise way to say it ("in your compilation" is one attempt but is probably confusing for novices).